<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetaRetencaoEstruturada</class>
 <widget class="QWidget" name="MetaRetencaoEstruturada">
  <property name="windowTitle">
   <string>Meta Retenção Estruturada</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_Periodo">
     <property name="minimumSize">
      <size>
       <width>237</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Período</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDateEdit" name="inicioPeriodo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="showGroupSeparator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::DaySection</enum>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="finalPeriodo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="showGroupSeparator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::DaySection</enum>
        </property>
        <property name="displayFormat">
         <string>dd/MM/yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_Acoes">
     <property name="minimumSize">
      <size>
       <width>257</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Ações</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="botaoProcessar">
        <property name="text">
         <string>Processar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botaoDetalhes">
        <property name="text">
         <string>Detalhes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botaoImprimir">
        <property name="text">
         <string>Imprimir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QFrame" name="frameDados">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frameGraphicPlot">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QSlider" name="girarEtiquetas">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Girar etiquetas &lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;&amp;quot;&lt;/span&gt;&lt;span style=&quot; font-weight:600; font-style:italic; color:#0055ff;&quot;&gt;Responsáveis pela Seleção&lt;/span&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;&amp;quot;&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="botaoSalvarScreenshot">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Salvar &lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;screenshot&lt;/span&gt; da plotagem do gráfico.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="thema.qrc">
           <normaloff>:/images/save_diskette.png</normaloff>:/images/save_diskette.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QCustomPlot" name="customPlot" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>inicioPeriodo</tabstop>
  <tabstop>finalPeriodo</tabstop>
  <tabstop>botaoProcessar</tabstop>
  <tabstop>botaoDetalhes</tabstop>
  <tabstop>botaoImprimir</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>girarEtiquetas</tabstop>
 </tabstops>
 <resources>
  <include location="thema.qrc"/>
 </resources>
 <connections/>
</ui>
